src/core/inlines.js

```diff
@@ -61,8 +61,8 @@ const l10n = getIntlData(localizationStrings);
 // TODO: Replace (?!`) at the end with (?:<!`) at the start when Firefox + Safari
 // add support.
 const inlineCodeRegExp = /(?:`[^`]+`)(?!`)/; // `code`
-const inlineIdlReference = /(?:{{[^}]+}})/; // {{ WebIDLThing }}
-const inlineVariable = /\B\|\w[\w\s]*(?:\s*:[\w\s&;<>]+)?\|\B/; // |var : Type|
+const inlineIdlReference = /(?:{{[^}]+\?*}})/; // {{ WebIDLThing? }}
+const inlineVariable = /\B\|\w[\w\s]*(?:\s*:[\w\s&;<>]+\??)?\|\B/; // |var : Type|
```

Comment:
This change appears to be leaked from https://github.com/w3c/respec/pull/3627

Reply:
oh woopsy. Will merge that first.